src/style.css

```
@@ -0,0 +1,80 @@
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+}
+
+.box {
+  display: flex;
+  background-color: #000;
+  flex-direction: column;
+}
+
+.block {
+  height: 300px;
+  min-width: 300px;
+  line-height: 300px;
+  text-align: center;
+}
+
+.first {
+  background-color: rgba(255, 0, 0, 100%);
+}
+
+.second {
+  background-color: rgba(255, 0, 0, 80%);
+}
+
+.third {
+  background-color: rgba(255, 0, 0, 60%);
+}
+
+.fourth {
+  background-color: rgba(255, 0, 0, 40%);
+}
+
+.fifth {
+  background-color: rgba(255, 0, 0, 20%);
+}
+
+.sixth {
+  background-color: rgba(255, 0, 0, 0);
+}
+
+@media (min-width: 600px) {
+  .box {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .block {
+    flex-grow: 1;
+  }
+
+  .sixth {
+    order: 3;
+  }
+  
+  .fifth {
+    order: 2;
+  }
+
+  .third {
+    order: 1;
+  }
+}
+
+@media (min-width: 900px) {
+  .fourth {
+    order: 5;
+  }
+
+  .fifth {
+    order: 4;
+  }
+
+  .block {
+    width: calc(100vw / 3);
```

```suggestion
    flex-basis: 33,3%;
```
Давай використаємо flex-basis